zedd-app/src/PlatformState.ts

```diff
@@ -54,6 +56,12 @@ export class PlatformState {
   @observable
   private _tasks: Task[] = []
 
+  @observable
+  private _taskActivities: TaskActivity[] = []
+
+  @observable
+  private _platformIntegration: PlatformIntegration
```

Comment:
```suggestion
  private platformIntegration: PlatformIntegration
```
Bitte umbennen _platformIntegration zu platformIntegration und @observable entfernen.